.github/workflows/test_s3_minio.yml

```diff
@@ -62,6 +59,7 @@ jobs:
           python -V
           which python
           pip install -e .
+          pytest
```

Comment:
will running the regular tests inside the minio container be any different than normally? We already run them once via `run-test-push.yml` workflow

Reply:
The tests don't run in the minio container (and they shouldn't - S3/minio should be a black box). The container runs in the background.

Running the integration tests here is very different from run-test-push.yml, because we're using S3 storage. We could skip the unit tests here, which should be the same in each case. Unsure what would be the best way to do that though, other than moving integration tests to an integration subdirectory.